network/packet-filtering-firewalls.md

```diff
@@ -110,3 +169,17 @@ $ firewall-cmd --reload
 ```
 
 Restarting `dockerd` daemon inserts the interface into the `docker` zone.
+
+## Docker and ufw
+
+[Uncomplicated Firewall](https://launchpad.net/ufw){: target="_blank" rel="noopener"}
+(ufw) is a frontend that ships with Debian and Ubuntu,
+and it lets you manage firewall rules. Docker and ufw use iptables in ways
+that make them incompatible with each other.
+
+When you publish a container's ports using Docker, traffic to and from that
+container gets diverted before it goes through the ufw firewall settings.
+Docker routes container traffic in the `nat` table, which means that packets
+are diverted before it reaches the `filter` table that ufw uses. Packets are
```

Comment:
I think people are more used to talk about the INPUT/OUTPUT chains than the filter table. Moreover, the current wording is not 100% accurate, because NATed packets go through the FORWARD chain located in the filter table.
 
```suggestion
are diverted before it reaches the `INPUT` and `OUTPUT` chains used by ufw. Packets are
```